docs/bigquery-gettingstarted.md

```diff
@@ -0,0 +1,150 @@
+![Setup Complete](images/09_setup_complete.jpg)
+
+## Understanding how the tables are structured
+So, now you have access! But what do you have access to?
+
+Most of the analysis we do is using tables in the httparchive.runs and httparchive.har folders. If you expand httparchive.runs, you'll notice that there are tables for `pages`, `requests`, `pages_mobile` and `requests_mobile`. There are sets of these tables for each HTTP Archive run dating back since 2010. And scrolling to the bottom of the folder, you'll notice that there are views that start with `latest_`. These contain the most recent data that is loaded.
+![HTTP Archive Runs Tables](images/httparchive_runs_tables.jpg)
+
+In order to understand what each of these tables contain, you can click on the table name and view the details. For example, after clicking on the 2017_08_15_pages table you can see the schema. Clicking `Details` tells you some information about the table, such as it's size and the number of rows. Clicking `Preview` shows an example of some data from the table.
+![Exploring Tables](images/exploring_data_pages_tables.jpg)
+
+
+Some of the types of tables you'll find useful when getting started are:
+### httparchive.runs Tables:
+* yyyy_mm_dd_pages:
+    * Each row contains details about a single page including timings, # of requests, types of requests and sizes.
+    * Information about the page load such # of domains, redirects, errors, https requests, etc
+    * Summary of different caching parameters
+    * CDN indicator
+    * Each page URL is associated with a "pageid"
+* yyyy_mm_dd_requests:
+    * Every single object loaded by all of the pages.
+    * Each object has a requestid and a pageid.  The pageid can be used to JOIN the _pages table.
+    * Information about the object, and how it was loaded
+    * Contains some response headers for each object
+    * These tables are very large (47GB as of Aug 2017)
+* yyyy_mm_dd_pages_mobile and yyyy_mm_dd_requests_mobile:
+    * Same as above, buf for mobile pages.
+ 
+### httparchive.har Tables:
+* yyyy_mm_dd_chrome_pages:
+    * HAR extract for each page url
+    * Table contains a url and a JSON-encoded HAR file for the document.
+    * These tables are large (~3GB as of Aug 2017)
+* yyyy_mm_dd_chrome_requests:
+    * HAR extract for each resource
+    * Table contains a document url, resource url and a JSON-encoded HAR extract for each resource
+    * These tables are very large (~168GB as of Aug 2017)
+* yyyy_mm_dd_chrome_requests_bodies:
+    * HAR extract containing response bodies for each request
+    * Table contains a document url, resource url and a JSON-encoded HAR extract containing the first 2MB of each response body.   
+    * payloads are truncated a 2MB, and there is a column to indicate whether they payload was truncated.
+    * These tables are very large (~760GB as of Aug 2017)
+* yyyy_mm_dd_chrome_lighthouse:
```

Comment:
LH only runs on mobile, so these tables are just an artifact full of null rows. I try to delete them but they come back every crawl :p